tools/release/common.sh

```diff
@@ -38,15 +42,21 @@ find_last_release() {
   export LAST_RELEASE_VERSION
 }
 
-# Find the previous release. If the last release is a stable release then it only takes in account stable releases
-# (the previous release of a stable release must be another stable release).
+# Previous release is one version before last release.
+# If last release is an rc, previous release can be any rc or non-rc.
+# If last release is a stable (non-rc) release, previous release must be a non-rc release.
 find_prev_release() {
   find_last_release
 
+  ALL_RELEASE_TAGS=$(git tag --list 'mimir-[0-9]*')
+
+  # To sort rc version correctly we use sed to append non-rc version with a temporary suffix
+  SORTED_RELEASE_TAGS=$(echo "$ALL_RELEASE_TAGS" | $SED '/rc/b; s/\(.*\)/\1-xx.x/' | sort --version-sort | gsed 's/-xx.x//')
```

Comment:
Should this use `$SED` too?

```suggestion
  SORTED_RELEASE_TAGS=$(echo "$ALL_RELEASE_TAGS" | $SED '/rc/b; s/\(.*\)/\1-xx.x/' | sort --version-sort | $SED 's/-xx.x//')
```

Reply:
just tested the script, doesn't work on linux without this fix

Reply:
Thank you @charleskorn for catching that.